.gitignore

```diff
@@ -11,3 +11,7 @@
 .phpunit
 /phpunit.xml
 ###< symfony/phpunit-bridge ###
+
+###> lexik/jwt-authentication-bundle ###
+%CONFIG_DIR%/jwt/*.pem
```

Comment:
shouldnt this be `/config/jwt/*.pem` :thinking:

Reply:
It's from the recipe.

Reply:
on it :D

Reply:
https://github.com/symfony/recipes/pull/386

Reply:
@DevMDamien can you re-install the recipe once more. I believe Flex handles the placeholder now.